SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -825,6 +825,28 @@ public void PatchWarningNote(PatchWarningNoteRequest request)
             var review = PostWarningNoteReview(request);
             _databaseContext.WarningNoteReview.Add(review);
             _databaseContext.SaveChanges();
+
+            var dt = DateTime.Now;
+
+            var note = new WarningNoteCaseNote
+            {
+                FirstName = person.FirstName,
+                LastName = person.LastName,
+                MosaicId = person.Id.ToString(),
+                Timestamp = dt.ToString("dd/MM/yyyy H:mm:ss"),
+                Note = $"{dt.ToShortDateString()} | Warning Note | {((request.Status == "closed") ? "Warning note against this person ended" : "Warning note against this person reviewed")}",
+                FormNameOverall = "API_WarningNote",
+                FormName = (request.Status == "closed") ? "Warning Note Ended" : "Warning Note Reviewed",
+                WarningNoteId = warningNote.Id.ToString(),
+                WorkerEmail = request.ReviewedBy
+            };
+
+            var caseNotesDocument = new CaseNotesDocument
+            {
+                CaseFormData = JsonConvert.SerializeObject(note)
+            };
+
+            _ = _processDataGateway.InsertCaseNoteDocument(caseNotesDocument).Result;
```

Comment:
Why do we assign a `discard` variable here since this method is a `void` return? 
Should we be returning this?

Reply:
The method `InsertCaseNoteDocument` returns `Task<string>` and not `void`, so we need to assign the result of the method to a variable. As we don't need the variable, it's assigned to a `discard`.